packages/shared-ux/page/analytics_no_data/src/services.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext } from 'react';
+import { Observable } from 'rxjs';
+
+type DataView = unknown;
+
+interface DataViewEditorOptions {
+  onSave: (dataView: DataView) => void;
+}
+
+/**
+ *
+ */
+export interface Services {
```

Comment:
nit: I find it strange to call this `Services` as it mixes methods (`openDataViewEditor`) with flags (`canAccessFleet)`. With those kind of interfaces I usually use `Context` which englobes different data type.

And then I would have probably structured it like this 😊 

```ts
interface Context {
  currentAppId$: Observable<string | undefined>;
  url: {
    addBasePath: (url: string) => string;
    navigateToUrl: (url: string) => Promise<void>;
  };
  authorization: {
    canAccesFleet: boolean;
    canCreateNewDataView: boolean;
  };
  state: {
    hasUserDataView: () => Promise<boolean>;
    hasDataView: () => Promise<boolean>;
    hasESData: () => Promise<boolean>;
  };
  dataView: {
    openEditor: (options: DataViewEditorOptions) => () => void;
  };
  docs: {
    kibanaGuideDocLink: string;
    dataViewsDocLink: string;
  };
  chrome: {
    setIsFullscreen: (isFullscreen: boolean) => void;
  }
}
```

Reply:
The reason I structured it flatly was because it's going to go away.

Since this component doesn't use any of these service methods, once the other components move to packages, this entire collection gets reduced and eventually deleted.

I need to add comments to that effect, though.